crates/pop-cli/src/commands/new/contract.rs

```diff
@@ -231,19 +231,4 @@ mod tests {
 		command.execute().await?;
 		Ok(())
 	}
-
-	#[tokio::test]
-	async fn test_new_contract_template_command_execute() -> Result<()> {
```

Comment:
This is tested in integration tests: https://github.com/r0gue-io/pop-cli/blob/main/crates/pop-cli/tests/contract.rs#L258